tests/integration/test_client.py

```diff
@@ -57,3 +67,65 @@ async def test_client_start_transcribe_stream_bad_request(self, client):
                 media_sample_rate_hz=9999999,
                 media_encoding="pcm",
             )
+
+    @pytest.mark.asyncio
+    async def test_client_start_transcribe_stream_bad_header(self, client):
+        with pytest.raises(BadRequestException):
```

Comment:
So if this is only service side, I don't think we need to test that. It may change at a later date and is already included in the documentation for the parameters.